json.md

```diff
@@ -0,0 +1,357 @@
+---
+layout: page
+title: JSON and serialization
+permalink: /json/
+---
+
+It is hard to think of a mobile app that doesn't need to communicate with a web
+server at some point. When making network-connected apps, the chances are that
+we need to consume some good old JSON, sooner or later.
+
+In this tutorial, we look into ways of using JSON with Flutter. We go over what
+JSON solution to use in different scenarios and why.
+
+* TOC Placeholder
+{:toc}
+
+## Which JSON serialization method is right for me?
+
+There isn't a one size fits all JSON solution for every project. 
```

Comment:
Add a short paragraph here:

This article covers two general strategies for working with JSON:

* Manual serialization and deserialization
* Automated serialization and deserialization via code generation